tests/ui/async-await/in-trait/return-not-existing-type-wrapping-rpitit.rs

```diff
@@ -0,0 +1,20 @@
+// edition:2021
+// [next] compile-flags: -Zlower-impl-trait-in-trait-to-assoc-ty
+// revisions: current next
+
+#![feature(return_position_impl_trait_in_trait)]
+#![feature(specialization)]
+#![allow(incomplete_features)]
+
+trait Foo {
+    fn bar() -> Wrapper<impl Sized>;
```

Comment:
Can you rename this `Wrapper` to be something more clear that it's missing?

```suggestion
    fn bar() -> Missing<impl Sized>;
```